Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyRowWidget.cpp

```diff
@@ -1755,7 +1755,7 @@ namespace AzToolsFramework
             return -1;
         }
 
-        for (int index = 0; index < GetParentRow()->GetChildRowCount(); index++)
+        for (AZ::u32 index = 0; index < GetParentRow()->GetChildRowCount(); index++)
```

Comment:
question: The return type of this function is ```int``` and ```index``` to returned on line1762. Will this not need an ```aznumeric_cast``` there to convert from ```AZ::u32``` to ```int```?